frontend/src/components/Layout/AppBar/AppBar.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <v-app-bar
-    :class="useResponsiveClasses('app-bar-safe-zone')"
+    class="app-bar-safe-zone"
     :elevation="transparentAppBar ? 0 : 3"
-    :color="transparentAppBar ? 'transparent' : undefined">
+    :color="transparentAppBar ? 'transparent' : undefined"
+    density="compact"
+    flat>
```

Comment:
Why?

Reply:
The flat property removes the default drop shadow (which displays as if it had `elevation` of 4. This won't matter until we remove the elevation property though. I'm going to look into using the app-bar extension as you pointed out though.